map/read-geopackage/src/main/java/com/esri/samples/read_geopackage/ReadGeoPackageSample.java

```diff
@@ -54,7 +41,7 @@ public void start(Stage stage) {
       // create stack pane and application scene
       StackPane stackPane = new StackPane();
       Scene scene = new Scene(stackPane);
-      scene.getStylesheets().add(getClass().getResource("/style.css").toExternalForm());
+      scene.getStylesheets().add(getClass().getResource("/css/style.css").toExternalForm());
```

Comment:
I think the old one (just `style.css`) is correct.

Reply:
I couldn't actually spot anything that uses this stylsheet reference, so I went ahead and removed it.